client/jest.config.json

```diff
@@ -13,7 +13,11 @@
     "build",
     "src/index.tsx",
     "src/AppProvider.tsx",
-    "src/store/store.ts"
+    "src/store/store.ts",
+    "src/util/queries/__generated__/",
+    "src/RelayEnvironment.ts",
```

Comment:
The file was generated during the Relay installation. It only has had minor changes to use Axios instead of fetch, and will later get authentication headers. I assumed it wouldn't make sense to test since it's mostly generated. What do you think? @prasadtalasila

Reply:
As I understand, the generated skeleton is being manually modified, so it makes sense to test it.